docs/choices-and-challenges/Choices and Challenges.md

```
@@ -1301,7 +1301,146 @@ Desuden:
+    - gratis version har begrænsninger og kræver ekstern konto
+    - bundler-audit og brakeman er allerede integreret og dækker hhv. dependency og code security
+    - snyk ville tilføje et dashboard, alerts og PR-kommentarer, hvor bundler-audit rapporterer i terminal og lader
+      pipeline fejle ved sårbarheder
+  
+**Implementering:**
+
+```markdown
+
+```
+
+**Rationale:**
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)